### DISCUSSION TITLE
Run application updates without the extra approval window

### DISCUSSION THREAD

**Discussion by CoreyJ87** [tteck/Proxmox#1893] [Q&A] [answered]
I am running cronicle. I want to be able to run say for instance this url [https://github.com/tteck/Proxmox/raw/main/ct/grafana.sh](https://github.com/tteck/Proxmox/raw/main/ct/grafana.sh) inside the LXC to update grafana. Currently it will blow up because it opens that extra window where you confirm you want to upgrade grafana and then pops back to the regular terminal. Is there a way to pass in a parameter to the script or something so it does not ask for approval and just run it? I just didnt want to have to go grab all the urls every time I want to upgrade all of the LXC apps I have. Wanted to just run the job and have it fire off that script headless basically. Thanks

**Comment by tteck** [comment, marked as answer]
Not at this time.
